ip.cc:34:19: error: 'srand' was not declared in this scope with ports-gcc
Index: libs-external/USI++/src/ip.cc
--- libs-external/USI++/src/ip.cc.orig
+++ libs-external/USI++/src/ip.cc
@@ -16,6 +16,7 @@
 #include <iostream>
 #include <string.h>
 #include <errno.h>
+#include <stdlib.h>
 #include <new>
 #include <vector>
 
